Transporter.MSSQLAdapter/Services/Interim/Implementations/InterimService.cs

```diff
@@ -55,16 +58,22 @@ public async Task DeleteAsync(IMsSqlInterimSettings settings, IEnumerable<dynami
 
         private async Task<string> GetInterimQueryAsync(IMsSqlInterimSettings settings)
         {
+            var timeDifferenceThreshold = GetTimeDifferenceThreshold();
             var sqlOptions = settings.Options;
             var query = new StringBuilder();
             query.AppendLine($"UPDATE TOP ({sqlOptions.BatchQuantity}) {sqlOptions.Schema}.{sqlOptions.Table}");
             query.AppendLine("SET Lmd=GETDATE() OUTPUT inserted.Id");
             query.AppendLine($"WHERE DataSourceName='{sqlOptions.DataSourceName}'");
-            query.AppendLine($"AND DATEDIFF(minute, lmd, GETDATE()) > 5");
+            query.AppendLine($"AND DATEDIFF(minute, lmd, GETDATE()) > {timeDifferenceThreshold}");
 
             return await Task.FromResult(query.ToString());
         }
 
+        private int GetTimeDifferenceThreshold()
+        {
+            return _configuration.GetSection(Constants.TimeDifferenceThreshold).Exists() ? _configuration.GetValue<int>(Constants.TimeDifferenceThreshold)  : 5;
```

Comment:
I am not sure if it is because of GitHub UI, but if there is extra space here, can you delete it?

Reply:
Extra space removed